apps/prairielearn/assets/scripts/instructorQuestionSettingsClient.ts

```diff
@@ -40,6 +41,27 @@ onDocumentReady(() => {
     },
   });
 
+  new TomSelect('#tags', {
+    valueField: 'name',
+    searchField: ['name', 'description'],
+    plugins: ['dropdown_input', 'remove_button'],
+    render: {
+      option(data: Tag) {
+        return html`
+          <div>
+            ${TagBadge(data)}
+            <div>
+              <small class="text-muted">${data.description}</small>
+            </div>
+          </div>
+        `.toString();
+      },
+      item(data: Tag) {
+        return html`<span class="badge color-${data.color} m-1">${data.name}</span>`.toString();
```

Comment:
We only need a margin on one side.

```suggestion
        return html`<span class="badge color-${data.color} mr-1">${data.name}</span>`.toString();
```